src/neo/Network/P2P/Payloads/OraclePayload.cs

```diff
@@ -0,0 +1,116 @@
+using Neo.IO;
+using System.IO;
+using Neo.Cryptography;
+using Neo.Cryptography.ECC;
+using Neo.Persistence;
+using System;
+using System.Linq;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class OraclePayload : IVerifiable
+    {
+        public byte[] Data;
+        //public uint OracleValidatorIndex;
+        public ECPoint OraclePub;
+        public Witness Witness;
+
+        public int Size =>
+            Data.GetVarSize() + //Data
+            OraclePub.Size +    //Oracle Public key
+            Witness.Size;       //Witness + 1 ??
```

Comment:
It's recommended to use cache like in `Transaction`, as `Size` will cost a lot of time.
https://github.com/neo-project/neo/blob/bf16df9d5a090dfcaff90d7c9d5454cf5ad09101/src/neo/Network/P2P/Payloads/Transaction.cs#L112-L127